go.mod

```diff
@@ -1,14 +1,18 @@
 module github.com/alibaba/opentelemetry-go-auto-instrumentation
 
-go 1.18
+go 1.21
 
 require (
 	github.com/dave/dst v0.27.3
-	golang.org/x/mod v0.8.0
+	go.opentelemetry.io/otel v1.28.0
```

Comment:
I suspect they are unnecessary dependencies as project's go.mod should not introduce otel packages, but it's up to you whether to keep these dependencies or not

Reply:
In opensource version, we will let users introduce ot dependency manually